src/FamilyHubs.Referral.Api/Endpoints/MinimalReferralEndPoints.cs

```diff
@@ -19,6 +22,25 @@ public class MinimalReferralEndPoints
 {
     public void RegisterReferralEndPoints(WebApplication app)
     {
+        app.MapPost("api/referralwithnotification", [Authorize(Roles = RoleGroups.LaProfessionalOrDualRole)] async ([FromBody] ReferralDataDto request, CancellationToken cancellationToken, ISender _mediator) =>
+        {
+            CreateReferralCommand referralcommand = new(request.ReferralDto);
+            long referralId  = await _mediator.Send(referralcommand, cancellationToken);
+
+            if (referralId > 0)
+            {
+                CreateUserAccountsCommand useraccountcommand = new(request.UserAccounts);
+                await _mediator.Send(useraccountcommand, cancellationToken);
```

Comment:
The referral has a reference to the UserAccount so should not the creation of the UserAccount be before the creation of the Referral?

Reply:
Should not really matter